source/documentation/deploying_services/elasticsearch.md

```diff
@@ -218,11 +218,11 @@ Aiven provide a Kibana user interface through which you can explore the data in
     cf ssh -L 4430:HOSTNAME_FROM_STEP_1:443 NAME_OF_AN_APP_YOU_CAN_SSH_TO
     ```
 
-    Elasticsearch instances are only accessible from the PaaS. This sets up a tunnel so that you can get access, similar to how we use [Conduit](https://github.com/alphagov/paas-cf-conduit) elsewhere.
+    Elasticsearch instances are only accessible from GOV.UK PaaS. This sets up a tunnel so that you can get access, similar to how we use [Conduit](https://github.com/alphagov/paas-cf-conduit) elsewhere.
```

Comment:
`from GOV.UK PaaS Infrastructure`?

Reply:
I think we say "from GOV.UK PaaS" elsewhere. Consistency over accuracy here, I reckon.